pages/instructorIssues/instructorIssues.ejs

```diff
@@ -29,112 +30,156 @@
         </div>
       </div>
 
-      <div class="card">
-        <div class="card-header bg-primary text-white">Issues</div>
+      <div class="modal fade" id="filterHelpModal" tabindex="-1" role="dialog" aria-labelledby="filterHelpModalLabel">
+        <div class="modal-dialog modal-lg" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h4 class="modal-title" id="filterHelpModalLabel">Filter help</h4>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              <p>Issues can be filtered and searched in a variety of ways. Filtering
```

Comment:
Also say that you can just enter a plain string to search for the text? Maybe with an example of doing this?

Reply:
Yep, good catch.